packages/beacon-node/src/chain/validation/attestation.ts

```diff
@@ -586,12 +586,9 @@ export async function getShufflingForAttestationVerification(
 
   const shuffling = await chain.shufflingCache.get(attEpoch, shufflingDependentRoot);
   if (shuffling) {
-    // most of the time, we should get the shuffling from cache
-    chain.metrics?.gossipAttestation.shufflingCacheHit.inc({caller: regenCaller});
```

Comment:
why removing these metrics? it's pretty important for attestation performance

Reply:
I move the metrics all into the ShufflingCache and its being `inc` inside the `.get`. Do you want me to add a 'caller` to the metric so we know where the `.get` call came from?  I copied that metric over with the other `shufflingCache` metrics to not break the dashboard and was going to do a follow up PR that updates and adds some new panels to the dashboards.

https://github.com/ChainSafe/lodestar/blob/mkeil/shuffling-refactor/packages/beacon-node/src/metrics/metrics/lodestar.ts#L1280

Reply:
yes need to add caller to `ShufflingCache.get()` and update dashboard in that case

Reply:
I put these back so they are there with attestation performance.  I will keep all the shuffling cache specific metrics together.